e2e/dashboards-suite/dashboard-public-create.spec.ts

```diff
@@ -114,7 +114,9 @@ e2e.scenario({
     e2e.pages.ShareDashboardModal.PublicDashboard.EnableSwitch().click({ force: true });
 
     // Save configuration
+    e2e().intercept('/api/dashboards/uid/ZqZnVvFZz/public-config').as('save');
     e2e.pages.ShareDashboardModal.PublicDashboard.SaveConfigButton().click();
+    e2e().wait('@save');
```

Comment:
I've never written an e2e, but is this something we should always do after making a request?

Reply:
Only when the next assertions could fail because depends on this request result.